src/components/Markdown.svelte

```diff
@@ -13,20 +13,19 @@
         start ? `start="${start}"` : ""
       } class="mzp-u-list-styled">${body}</${outerEl}>`;
     },
+    html(html) {
+      return `<code>${html.replace("<", "&lt;")}</code>`;
+    },
```

Comment:
Along with replacing `<`, should we also replace `>`?
```
  const mapChar = {
    "<": "&lt;",
    ">": "&gt;",
  };
```

```suggestion
    html(html) {
      return `<code>${html.replace(/<|>/g, (char) => mapChar[char])}</code>`;
    },
```

Reply:
I'm pretty sure this isn't necessary. `>` by itself should parse fine in almost all cases: there might be one or two cases where weird things might happen, but I think they're unlikely enough that I'd rather keep this code simple.